ui/package.json

```diff
@@ -40,9 +40,10 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-object-rest-spread": "^7.5.5",
+    "@babel/plugin-transform-block-scoping": "^7.10.4",
     "@ember/optional-features": "^0.7.0",
     "@hashicorp/structure-icons": "^1.3.0",
-    "@storybook/ember-cli-storybook": "meirish/ember-cli-storybook#6bd58326d8c21e986d390b541ae5e49089d61b93",
+    "@storybook/ember-cli-storybook": "^0.2.1",
```

Comment:
note that we no longer need to use the forked version of `ember-cli-storybook` since the updated version includes the bugfix we were waiting for!